documentation/src/main/asciidoc/topics/con_shared_cache_stores.adoc

```diff
@@ -6,16 +6,14 @@ By default, cache stores are local (`shared="false"`).
 
 * Local cache stores are unique to each node; for example, a file-based cache store that persists data to the host filesystem.
 +
-Local cache stores can fetch state and purge on startup to avoid loading stale entries from persistent storage.
+Local cache stores can use purge on startup to avoid loading stale entries from persistent storage.
```

Comment:
"can use purge" could change to "can purge data"

I'm pretty sure purge is transitive so "purge data" would be the correct form here.

Reply:
The problem is purge is not being used a verb here but rather the term is "purge on startup". How do you point out terms like this normally? Should I italicize it?

Reply:
Wrapping in double quotes like you have there is the way to go for a known phrase. Thanks.